### PR TITLE
Create mlhajipur.json

### PR DESCRIPTION
**Pull request by Anushka-codergirl**
## Types of changes

- [x] New Subdomain Request
- [x] Modification to Existing Subdomain
- [x] Github Repository Update
- [x] Other

## Checklist:

- [x] This change will not affect the existing subdomains.
- [x] I have checked the existing subdomains and ensured that my new subdomain nor modification is not a duplicate.

### CODE REVIEW

**Comment by BRAVO68WEB** [comment]
Hey @Anushka-codergirl , thank you for making a pull request to our project.

There are few issues, in your request. 

1. Create a Issue [here](https://github.com/ml-comm/ddns/issues/new?assignees=&labels=request&projects=&template=REQUEST.yml&title=%5BREQUEST%5D%3A+) about your request.
2. We are unable to understand what domain you want to link with?

**Comment by BRAVO68WEB** [comment]
Please start by creating a Issue at [https://github.com/ml-comm/ddns/issues/new?assignees=&labels=request&projects=&template=REQUEST.yml&title=%5BREQUEST%5D%3A+](here)

**Review by BRAVO68WEB — CHANGES_REQUESTED** [review]
Approval Issue not found; Invalid Config

**Comment by BRAVO68WEB** [comment]
@smanitech

**Comment by Anushka-codergirl** [comment]
Done👍

**Review by Ba3a-G — CHANGES_REQUESTED** [review]
Hey @Anushka-codergirl,
We are unable to understand your request. 
Here's what you need to write in each of the parameters:

- owner
  - username: _your GitHub username_
  - email: _your Organisation's email_
- target
  - CNAME
    - name: _your desired subdomain_
    - value: _your existing hosted website_

Also, you need to change the filename from _mlhajipur_ to _hajipur_.
Please RTFM [here](https://ddns.mlcommunity.dev/format.html).

**Comment by Anushka-codergirl** [comment]
I got it!

File: domains/hajipur.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+    "owner": {
+        "username": "Anushka-codergirl",
+        "email": "hajipur@mlcommunity.dev"
+    },
+
+    "target": {
+        "CNAME": {
+            "name": "hajipur",
+            "value": "Anushka-codergirl.github.io"
```
Comment (Ba3a-G) [inline domains/hajipur.json:12]: Hey, the website at Anushka-codergirl.github.io seems to be a personal portfolio. We can provide subdomains only to community websites.

**Review by Ba3a-G — CHANGES_REQUESTED** [review]

**Comment by BRAVO68WEB** [comment]
@Anushka-codergirl, can you let us know, where is your website repo on github?

**Comment by Anushka-codergirl** [comment]
https://github.com/Anushka-codergirl/MLHajipur - I will push the code once it's ready!

**Comment by BRAVO68WEB** [comment]
Got it, we are closing this PR for now. Recreate the pr once the website is built. We will assist you that time.

**Pull request closed without merging**